docs/ad/analysis/index.rst

```diff
@@ -0,0 +1,32 @@
+.. include:: /ad/ad_attention.inc
+
+.. py:currentmodule:: plasmapy.analysis
+
+.. _analysis:
+
+==============================
+Analyses (`plasmapy.analysis`)
```

Comment:
```suggestion
Analysis tools (`plasmapy.analysis`)
```

Reply:
I wanted to keep "Analyses" because it better paired with "Diagnostics" in the sidebar and TOC.  I honestly prefer "Analysis Tools" as a heading, but felt the Analyses-Diagnostics pair look and feel in the sidebar and TOC beat out my preference.

Reply:
To be honest, I think I still prefer "Analysis tools" despite the nice pairing in the sidebar: it's more explicit. "Analyses" makes me think of something I might do using a framework, not the toolset I'd expect to find in that framework. Closer to "use cases", probably, just stated weirdly.

I second Nick's suggestion here.